### PR TITLE
makes BASE_URL available in bash

### PR DESCRIPTION
**Pull request by funkyfuture**

### CODE REVIEW

**Comment by samos123** [comment]
Not sure what's the use case. But code looks fine to me . You could also consider putting it in /etc/profile or /etc/environment depending on the usecase.

**Comment by funkyfuture** [comment]
`/etc/environment` is related to pam, no idea if that plays a role in a container. it's for debugging. maybe it can be of use for anyone.

i added some further commits.

File: entrypoint.sh [hunk 1]
```diff
@@ -145,6 +145,9 @@ fi
 
 run_scripts pre-launch
 
+unset ADMIN_USER
+unset ADMIN_PASSWORD
```
Comment (samos123) [inline entrypoint.sh]: Nice job on catching this.

**Comment by samos123** [comment]
Looks good to me. Merge as you please :)

**Pull request merged**